eng/scripts/Language-Settings.ps1

```diff
@@ -23,7 +23,7 @@ function Get-AllPackageInfoFromRepo ($serviceDirectory)
   try
   {
     Push-Location $RepoRoot
-    python -m pip install "./tools/azure-sdk-tools[build]" -q -I
+    $installResult = python -m pip install "./tools/azure-sdk-tools[build]" -q -I
```

Comment:
Is it worth dumping this to the log in failure cases or something? If you don't ever plan to use the info then indicate that by using `$null =`